docs/current_docs/index.mdx

```diff
@@ -20,6 +20,8 @@ Dagger is an open-source runtime for composable workflows. It's perfect for syst
 
 - **Built-in Observability:** Full visibility into operations with tracing, logs, and metrics. Debug complex workflows and know exactly what's happening.
 
+<p align="center"><img src="img/cloud-trace.gif" width="60%"></img>
```

Comment:
```suggestion
![Dagger Trace](/img/cloud-trace.gif)
```

I'm generally not in favour of mixing HTML with Markdown unless there's a good reason. In this case, I think the suggestion above should achieve the required result.

Reply:
Side note: images which appear in the current docs content should be stored in static/img/current_docs (we can fix this separately, noting here for reference)

Reply:
Going ahead and pushing this change to fix the broken build